README.md

```diff
@@ -46,23 +46,29 @@ Note: if you prefer, you can use the pre-compiled binary found [HERE](https://gi
 
 #### Prerequisites
 
-You must first be able to build PHP 7 without including these extensions.  For help with doing this, see the [official PHP website][phpbuild] for building your own PHP on Windows.
+You must first be able to build PHP 7 without including these extensions. For help with doing this, see the [official PHP website][phpbuild] for building your own PHP on Windows.
 
 #### Compile the drivers
 
```

Comment:
Usually in our other READMEs the commands that a user is expected to run are in code blocks it makes it easier to read and c&p the commands and also keep things consistent